### DISCUSSION TITLE
Disable canvas for Excepted Selected Component

### DISCUSSION THREAD

**Discussion by Logesh-cartrabbit** [GrapesJS/grapesjs#6303] [Q&A]
In Grapesjs, I want to disable the entire canvas except for a particular selected Component how to acheive this? for Example if i loop throught  the entire wrapper it takes toomuch time for loading the components @artf

**Comment by artf** [comment]
You can make use of the `locked` property.
```js
// lock the wrapper
editor.getWrapper().set({ locked: true })
// unlock specific component
someComponent.set({ locked: false })
```

**Reply by Logesh-cartrabbit** [reply]
@artf  Actually the Locked Property Doesn't working i think but the property selectable to false working
